src/pr-info.ts

```diff
@@ -427,12 +435,26 @@ function makeJsonCheckerFromCore(requiredForm: any, ignoredKeys: string[]) {
     };
 }
 
-function usersSayReadyToMerge(comments: PR_repository_pullRequest_comments_nodes[], users: string[], sinceDate: Date) {
-    return comments.some(comment =>
-        comment
-        && users.includes(comment.author?.login || " ")
-        && comment.body.trim().toLowerCase().startsWith("ready to merge")
-        && (new Date(comment.createdAt)) > sinceDate);
+function latestComment(comments: PR_repository_pullRequest_comments_nodes[]) {
+    if (comments.length === 0) return undefined;
+    return comments.reduce((r, c) => r && Date.parse(r.createdAt) > Date.parse(c.createdAt) ? r : c);
+}
+
+function getMergeOfferDate(comments: PR_repository_pullRequest_comments_nodes[], abbrOid: string) {
+    const offer = latestComment(comments.filter(comment =>
+        sameUser("typescript-bot", comment.author?.login || "-")
+        && Comment.parse(comment.body)?.tag === "merge-offer"
```

Comment:
This capitalization really threw me off for a second, not offensive but was a tad suspicious

Reply:
Which one?  I don't mind changing...

(`Comment` is similar to `Comments`, if that's the one...)

Reply:
`Comment` I thought it was a `class`

Reply:
OK, I'll change it.